localstack/testing/pytest/snapshot.py

```diff
@@ -30,6 +27,7 @@ def pytest_configure(config: Config):
 @pytest.hookimpl
 def pytest_addoption(parser: Parser, pluginmanager: PytestPluginManager):
     parser.addoption("--snapshot-update", action="store_true")
+    parser.addoption("--snapshot-skip-all", action="store_true")
     parser.addoption("--snapshot-verify", action="store_true")
```

Comment:
are we using this flag at all anywhere?

Reply:
No, we don't :) Dominik thought it might be nice to have though. 

Default is enabled, single tests can be disabled by adding the marker `pytest.mark.skip_snapshot_verify` and this would be the option to disable all snapshot tests. 
What are your thoughts on that?

And we parse the flag in the `snapshot` fixture, if that was the question 🤔 
```
 sm = SnapshotSession(
       ...
       verify=False if request.config.option.snapshot_skip_all else True
)
```